Dockerfile

```diff
@@ -3,7 +3,7 @@
 # A container for the core semantic-search capability.
 #
 ######################################################
-FROM python:3.9.6-slim
+FROM python:3.10.10-slim
```

Comment:
I attempted to use 3.11 it caused additional breaking packages. 

I will try again now that you are using fastAPI instead of flask though if I remember right that was the issue however it may also cause issues with Roger.

Reply:
@braswent if it takes more than minor updates, i think it will be enough to create a task for this and handle it separately.